test/over_react/util/react_wrappers_test.dart

```diff
@@ -1101,3 +1101,12 @@ class PlainObjectPropsMap {
 class PlainObjectStyleMap {
   external get width;
 }
+
+/// Helper component that renders whatever you tell it to. Necessary for rendering components with the 'ref' prop.
+final RenderingContainerComponentFactory =
```

Comment:
This was only needed by this file, so I moved it here from `test/test_util/react_util.dart`